ui/src/adapters/api/schemas.ts

```diff
@@ -16,28 +16,40 @@ const apiSchemaParser = getStrictParser<ApiSchemaInput, ApiSchema>()(
   z.object({
     bigInt: z.string(),
     createdAt: datetimeParser,
+    description: z.string().nullable(),
     hash: z.string(),
     id: z.string(),
+    title: z.string().nullable(),
     type: z.string(),
     url: z.string(),
+    version: z.string().nullable(),
   })
 );
 
 export async function importSchema({
+  description,
   env,
   jsonLdType,
   schemaUrl,
+  title,
+  version,
 }: {
+  description?: string;
   env: Env;
   jsonLdType: JsonLdType;
   schemaUrl: string;
+  title?: string;
+  version?: string;
 }): Promise<Response<ID>> {
   try {
     const response = await axios({
       baseURL: env.api.url,
       data: {
```

Comment:
I think that this could be just:

```ts
      data: {
        description: description || null,
        schemaType: jsonLdType.name,
        title: title || null,
        url: schemaUrl,
        version: version || null,
      },
```

Reply:
Wouldn't that cause legit empty strings to become `null`?

<img width="411" alt="image" src="https://github.com/0xPolygonID/issuer-node/assets/15896177/c610ae80-1408-4ae4-b26e-62d38934d881">

Reply:
But this `title`, `description` and `version` fields are required right? We shouldn't send empty strings for them right?

Reply:
Nope none of them are required.